workbench-core/example/infrastructure/integration-tests/tests/datasets/accessPermissions.test.ts

```diff
@@ -232,24 +232,37 @@ describe('DataSets access permissions integration tests', () => {
           accessLevel: 'read-only'
         }
       });
-      await expect(dataSet.getAllAccess()).resolves.toMatchObject({
-        data: {
-          dataSetId: dataSetId,
-          permissions: [
-            {
-              identityType: 'GROUP',
-              identity: groupId,
-              accessLevel: 'read-write'
-            },
-            ...permissions,
-            {
-              identityType: 'USER',
-              identity: userId,
-              accessLevel: 'read-only'
-            }
-          ]
+      // using await expect(...).toMatchObject(...) requires permissions order to be
```

Comment:
you could use Jest's `arrayContaining()` method which just checks if the listed values are in the array.
```
toMatchObject({
...
permissions: expect.arrayContaining(<permissionsList>)
...
})
```